client/src/pages/Login/LoginForm/LoginForm.tsx

```diff
@@ -48,7 +48,7 @@ export default function Login({ handleSubmit }: Props): JSX.Element {
         <form onSubmit={handleSubmit} className={classes.form} noValidate>
           <TextField
             id="email"
-            label={<Typography className={classes.label}>E-mail address</Typography>}
+            // label={<Typography className={classes.label}>E-mail address</Typography>}
```

Comment:
We should remove this instead of commenting it out before merging.

Reply:
Got it!